### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by samsiegart**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+```
+
+## Connecting to User's Account (Keplr)
+
+*package.json:*
+```
+"@agoric/notifier": "^0.6.2",
+"@agoric/rpc": "^0.4.1-dev-7cf64bb.0",
+"@agoric/web-components": "0.10.1-dev-7cf64bb.0"
+```
```
Comment (turadg) [inline README.md:59]: ditto, most readers of the README shouldn't need these version numbers.

for simplicity we could just hold off on merging this PR until we've published these versions as full releases.

**Review by turadg — COMMENTED** [review]
This documentation is great.

I wonder if it should be in the `rpc' package, which doesn't have a README right now. All the examples do use rpc. And that way it will appear on NPM https://www.npmjs.com/package/@agoric/rpc

Maybe root has "UI KIt" and "Setup" and pointers to each package README which have the snippets.

That other README could be part of the PR publishing rpc 0.5.0. So this PR can stay as is until that's in main

File: README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+
+*app.ts:*
+```ts
+import {
+  makeAgoricChainStorageWatcher,
+  AgoricChainStoragePathKind as Kind
+} from '@agoric/rpc';
+
+const watcher = makeAgoricChainStorageWatcher(rpc, chainName);
```
Comment (dckc) [inline README.md:30]: I'm used to passing network access (`fetch`) in explicitly. I don't expect to change that here, but it's worth indexing under https://github.com/Agoric/agoric-sdk/issues/2160

**Comment by samsiegart** [comment]
@turadg I removed the dev npm versions and added some smaller snippets under the packages themselves. PTAL

**Review by turadg — APPROVED** [review]

**Pull request merged**